src/twisted/internet/tcp.py

```diff
@@ -656,7 +659,7 @@ def _connectDone(self):
             self.protocol.makeConnection(self)
 
 
-_NUMERIC_ONLY = socket.AI_NUMERICHOST | _AI_NUMERICSERV
+_NUMERIC_ONLY = socket.AI_NUMERICHOST | socket.AI_NUMERICSERV
 
 
 def _resolveIPv6(ip, port):
```

Comment:
Given that the logic has changed a bunch here it would be good to get a type annotation to ensure the signature has not changed.